mod_reforged/hooks/ui/screens/tooltip/tooltip_events.nut

```diff
@@ -13,7 +13,7 @@
 					{
 						id = 2,
 						type = "description",
-						text = ::Reforged.Mod.Tooltips.parseString("Characters gain experience as they or their allies slay enemies in battles. If a character has accumulated sufficient experience, he\'ll [level up|Concept.Level]. Experience for killing an entity is shared among all the characters who did damage to that entity, proportional to the damage dealt by each character. Additionally some experience is awarded to all members of a faction who did damage to that entity, proportional to the damage dealt by that faction.\n\nBeyond the " + ::Const.XP.MaxLevelWithPerkpoints + "th  character level, characters are veterans and gain perk points only every 4th level. They can still improve their attributes but the attribute gain per level is small.")
+						text = ::Reforged.Mod.Tooltips.parseString("Characters gain experience as they or their allies slay enemies in battles. If a character has accumulated sufficient experience, he\'ll [level up|Concept.Level].\n\nExperience gained from dying enemies is proportional to the fraction of the total damage dealt to that enemy by your company. Of this experience, " + ::Const.XP.XPForKillerPct + "% is shared proportionally between the brothers who did damage. The other " + (100 - ::Const.XP.XPForKillerPct) + "% is shared equally among all members of the company.\n\nBeyond the " + ::Const.XP.MaxLevelWithPerkpoints + "th  character level, characters are veterans and gain perk points only every 4th level. They can still improve their attributes but the attribute gain per level is small.")
```

Comment:
I suggest we add a global config variable for Veteran Perks and use that in this tooltip aswell as in the `rf_veteran_levels.nut`. Currently that variables value would be `4`
That doesn't to be part of this PR.